p2p/dial.go

```diff
@@ -77,7 +77,7 @@ var (
 	errAlreadyDialing   = errors.New("already dialing")
 	errAlreadyConnected = errors.New("already connected")
 	errRecentlyDialed   = errors.New("recently dialed")
-	errNotWhitelisted   = errors.New("not contained in netrestrict whitelist")
+	errNotAllowListed   = errors.New("not contained in netrestrict allowList")
```

Comment:
This sounds like a mouthfull.

Reply:
The error name could be `errNetRestrict`